packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js

```diff
@@ -4,4 +4,8 @@ import template from './overTheBox-configure.html';
 export default {
   controller,
   template,
+  bindings: {
+    orderHash: '<',
```

Comment:
Hmm this doesn't seems to be used no?

Reply:
No it's used into the file overTheBox-configure.html

Reply:
i don't see either where it's used can you link the line ?

Reply:
L43 of overTheBox-configure.html (which not change)